switchapi/es2k/lnw_v3/switch_pd_routing.c

```diff
@@ -1735,6 +1735,7 @@ switch_status_t switch_pd_ecmp_hash_table_entry(
     goto dealloc_resources;
   }
 
+  /* LNWv3 does not have this parameter
```

Comment:
Please use `#if 0` / `#endif` to disable code.

Commenting code out this way is fragile and less apparent to the reader. It makes the code harder to maintain.

Reply:
Fixed